apps/frontend/src/views/Results.vue

```diff
@@ -456,6 +473,16 @@ export default class Results extends Vue {
       this.eval_info = index;
     }
   }
+
+  // Expand all visible controls
+  expand_all() {
+    (this.$refs.controlTable as Vue & { expandAllEvaluations: () => void }).expandAllEvaluations();
+  }
+
+  // Toggles whether or not to show expand all button
+  set_show_expand_all(single_expand: boolean) {
+    this.show_expand_all_button = !single_expand;
+  }
```

Comment:
No need to hide the button, just make the button change the state of Single Expand.